extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java

```diff
@@ -185,6 +194,19 @@ void nativeImageConfiguration(
                         .reason(getClass().getName())
                         .constructors().methods().fields().build());
 
+        // register all liquibase.datatype.core.* data types
+        Set<String> classesAnnotatedWithDataTypeInfo = combinedIndex.getIndex().getAnnotations(DATA_TYPE_INFO_ANNOTATION)
+                .stream()
+                .map(AnnotationInstance::target)
+                .filter(at -> at.kind() == AnnotationTarget.Kind.CLASS)
+                .map(at -> at.asClass().name().toString())
+                .filter(s -> !s.endsWith("Snowflake"))
```

Comment:
Since there is [no support](https://github.com/quarkusio/quarkus/issues/41017) for snowflake in quarkus native right now, I saw no reason  to include classes for this purpose.

Reply:
It doesn't cost us much, I think I would rather remove the special case as if one day someone wants to create an extension, we will have to think about removing this.

Reply:
No problem, I'll remove the filter.

Reply:
Thanks. I made some checks and I think we can backport the whole thing. Thanks for this cleanup!